data_structures/src/data_request.rs

```diff
@@ -25,9 +25,19 @@ pub struct DataRequestPool {
     pub data_request_pool: HashMap<Hash, DataRequestState>,
     /// List of data requests that should be persisted into storage
     pub to_be_stored: Vec<DataRequestReport>,
+    /// Extra rounds for commitments and reveals
+    pub extra_rounds: u16,
 }
 
 impl DataRequestPool {
+    /// Create a new 'DataRequestPool' initialized with a number of extra rounds
+    pub fn new(extra_rounds: u16) -> Self {
+        let mut data_request_pool = DataRequestPool::default();
+        data_request_pool.extra_rounds = extra_rounds;
+
+        data_request_pool
+    }
```

Comment:
Expressions FTW!!!! :sunglasses:  
```suggestion
    pub fn new(extra_rounds: u16) -> Self {
        DataRequestPool {
            extra_rounds,
            ..Default::default()
        }
    }
```

Reply:
Take this only as a suggestion. For what it's worth, it has my approval either way.